charts/fission-all/values.yaml

```diff
@@ -479,6 +478,36 @@ prometheus:
 canaryDeployment:
   enabled: false
 
+
+authentication:
+  ## set this flag to true if you need authorization
+  ## for all route calls
+  ##
+  enabled: false
+  ## authUrlPath is the router path for
+  ## authentication endpoint, defaults to /auth/login
+  ## deafult '/auth/login'
+  ##
+  authUrlPath: /auth/login
```

Comment:
Just wondering if tomorrow if we consider an external auth provider? Would it work with these params?

Reply:
The auth URI  value is looked for in the env variable `FISSION_AUTH_URI`. If it is not set or if it is a empty string then it defaults to `/auth/login`